### ISSUE TITLE
Minimap icons not showing up

### ISSUE THREAD

**Issue by RoboHoboMK2** [AeroScripts/QuestieDev#1698]
## Bug description
Since version v5.4.1, I have periodically had to adjust the zoom one level on the minimap to make the minimap icons appear. For versions 5.5.1 and 5.5.0 the bug has become permanent (i.e. no longer periodical). I've run around twenty fresh addon installs with various Questie versions, each on a clean WoW Classic install, to ensure (somewhat) reliable data.

Curiously, the bug also seems to affect the GatherLite addon minimap icons.

I really hope this is something you can address.

## Screenshots
Did not find this necessary - just imagine a minimap free of all Questie icons.

## Questie version
Affects v5.4.1, 5.5.0 and 5.5.1

Which version of Questie are you using? You can find it by:
v5.4.1, as the bug is only intermittent here.

**Comment by BreakBB** [comment]
@RoboHoboMK2 "on a clean WoW Classic install". Does that mean you also deleted the SavedVariables where all the addon configuration is saved? We didn't have any report of such an issue before from other users.

Did you check [the FAQ](https://github.com/AeroScripts/QuestieDev/wiki/FAQ) and tried with all other addons disabled? Maybe there is a new conflict with another addon.

**Comment by RoboHoboMK2** [comment]
@BreakBB Yes, I deleted the SavedVariables both in the global account folder (...\World of Warcraft\_classic_\WTF\Account\[account_name]\SavedVariables) and in the character specific folders (...\World of Warcraft\_classic_\WTF\Account\[account_name]\[server_name]\[character_name])

I tried with all other addons disabled, with all other addons uninstalled (i.e. only the Questie addon in the "...\interface\addons"-folder). I also removed all other addons' SavedVariables data.

In short - I tested the various versions of Questie on completely fresh installs with no other addons or previous addon/game data present anywhere in the game's folder hierarchy.

I haven't a clue what causes the bug. All I know is that I'm able to reproduce it very reliably and with nearly 100% consistency. Only in version 5.4.1 do I get slight variation, with icons loading about 20% of the time.

**Comment by BreakBB** [comment]
This is very strange and I honstely don't have a real idea what could be causing it... Do you maybe have a lot of quests active which have to be loaded and is your PC a bit weak?

**Comment by RoboHoboMK2** [comment]
I'm afraid not.

The bug was consistent across 2 to 20 quests. My laptop is an Acer Nitro 7 with an i7 9750H, 1660ti, bleeding fast SSD and so on and so forth.

It is weird, aye. For now I'm fine using older versions though, so it's not a huge problem.

**Comment by BreakBB** [comment]
So it's not your PC :thinking:

Since it also affects GatherLite addon it could be connected to the underlying librarby we're using. But on the other hand this doesn't really make sense to me since you seem to be the only one with such a problem...

**Comment by BreakBB** [comment]
Related to #1762 and fixed with v5.7.2
